src/xdocs/beginning_development.xml

```diff
@@ -31,15 +31,21 @@
         <a href="https://help.github.com/articles/fork-a-repo/"> here</a><br/>
         3. Checkout the current source code from: https://github.com/you_user_name/checkstyle/<br/>
         by running
-        <source class="wrap-content">
-git clone git@github.com:you_user_name/checkstyle.git
+      </p>
+      <div class="wrap-content">
+        <source>
```

Comment:
Why the change from `<source>` to `<div><source>`?

Reply:
Because `<source>` must not have a `class` attribute.

Reply:
Why? Is there some place that states it is not valid? We use it already and have had no issues.

Reply:
Because the validation of the maven-site-plugin is disabled because of a bug that is fixed now.
If you switch on validation and do not do the changes I made, the pages will not validate.
I basically mostly did changes that were necessary to make the pages validate properly.
It worked until now because validation was turned of because of said bug and browsers are very lenient in parsing HTML and correcting errors in it to have a better experience for users that browse to pages that are acutally not valid.

Reply:
Ok thanks.
If there is no or minor change in display because of these 2 line changes then I am fine. Can you provide a screenshot?

Reply:
@Vampire , thanks a lot for your desire to help, I really appreciate this, but lets follow rules of our project, we will guide you. Sorry if extra work might be required.

first of all this is not a "doc:" minor change, please create issue on it and lets describe all reasons/nuances in issue, not a PR comments.

Reply:
It looks the same, screenshot is in the comment above I just made.

Reply:
Ah, ok, didn't know `doc:` is only for minor doc changes.
Well, in effect they actually **are** minor, but I'll open an issue later, gotta go now.

Reply:
http://rveach.no-ip.org/checkstyle/regression/reports/fix-invalid-doc/beginning_development.html#Before_development

There is a minor change here. Box shrunk compared to original. (I use firefox)
http://checkstyle.sourceforge.net/beginning_development.html#Before_development

I am ok with the shrinkage. It looks like excess space anyways that we didn't need.
I will mark this resolved.

Reply:
@Vampire 
> Because <source> must not have a class attribute.
> Why? Is there some place that states it is not valid?
> Because the validation of the maven-site-plugin is disabled 

Do you know how maven-site-plugin determines what children are acceptable and which aren't? Is it a fixed list built into it's code?

Reply:
> Do you know how maven-site-plugin determines what children are acceptable and which aren't? Is it a fixed list built into it's code?

It is the same children your IDE or any other proper XML editor allows:
![grafik](https://user-images.githubusercontent.com/325196/54170344-96e60f00-4476-11e9-910f-58c1b2cc4d26.png)

It is defined in the XDoc 2.0 XML schema referenced in the root tag of the file.